src/Sulu/Bundle/AdminBundle/Resources/js/containers/Form/stores/SchemaFormStoreDecorator.js

```diff
@@ -6,10 +6,14 @@ import metadataStore from './metadataStore';
 export default class SchemaFormStoreDecorator implements FormStoreInterface {
     @observable innerFormStore: ?FormStoreInterface;
 
-    constructor(initializer: (schema: RawSchema, jsonSchema: Object) => FormStoreInterface, formKey: string) {
+    constructor(
+        initializer: (schema: RawSchema, jsonSchema: Object) => FormStoreInterface,
+        formKey: string,
```

Comment:
shouldnt we add a `type` param now? later on we need to use another parameter order than the `metadataStore `

Reply:
Good catch, will add it.